app/experimenter/experiments/models.py

```diff
@@ -674,16 +674,22 @@ def _conditional_required_reviews_mapping(self):
         }
 
     def _default_required_reviews(self):
+        # Ref: #2325, request for specific order of reviews
```

Comment:
Eh I don't think you need to leave comments like this hanging around

Reply:
In general the only time I think I see things like this are if there's something that needs changing and you leave a TODO: #bugnumber, but in this case the git history will record the commit where that line was changed which will refer to the bug number so I don't think you need to leave a reference right in the code?

Reply:
right right completely forgot about git history